### PR TITLE
fix #86

### LINKED ISSUE #86: Can not support the input method.
Can only enter letters.

### PR DESCRIPTION
**Pull request by kid1412z**
#86 When using input method, the textarea will not be cleared. So every time input a new string it will append to the old ones. 
Adding this line fixed Chinese input method problem described above.

### CODE REVIEW

File: js/jquery.terminal.js [hunk 1]
```diff
@@ -1357,6 +1358,8 @@
                     } else {
                         if (command !== '' && position > 0) {
                             self['delete'](-1);
+                            var str = $('.clipboard').val()
```
Comment (jcubic) [inline js/jquery.terminal.js]: You forget semicolon.

File: js/jquery.terminal.js [hunk 2]
```diff
@@ -1357,6 +1358,8 @@
...
+                            $('.clipboard').val(str.substring(0, str.length - 1));
```
Comment (jcubic) [inline js/jquery.terminal.js]: If you want to clear the clipboard textarea why don't you just use `$('.clipboard').val('');`

Comment (jcubic) [inline js/jquery.terminal.js]: Sorry I thought tat you removing the whole clipboard. 
I think that the clipboard deleteting should be moved to delete method and it should take into account cursor position or better fake_mobile_entry should be renamed, maybe to fix_clipboard and the `if (is_touch) {` should be removed from that function, this should fix the problem with input method.

Comment (jcubic) [inline js/jquery.terminal.js]: I don't expect you to make those changes, just thinking out loud for improvements.

Comment (kid1412z) [inline js/jquery.terminal.js]: Yeah, it should be more graceful,when I was stucked I just debug into it and find the dirty but easy way out, did not pay attention to many other code around it. I d like to help if you want.

Comment (jcubic) [inline js/jquery.terminal.js]: You can remove the `if (touch) {` from [fake_mobile_entry](https://github.com/jcubic/jquery.terminal/blob/master/js/jquery.terminal-src.js#L934] change the name to `fix_textarea` and change all occurences of that function then you will be able to remove that substring from backspace and it should work. I'll be happy to merge if you do.

Comment (kid1412z) [inline js/jquery.terminal.js]: done, it work will on my mac chrome. But I did not test it on android. Thanks for you hint and thanks for your work of this project!

**Comment by jcubic** [comment]
Sorry I didn't notice that you modify jquery.terminal.js file, that file is created by `make`, based on jquery.terminal-src.js. Can you modify that file instead and run `make`, you will need to have instaled node and npm and run `npm install` before you run `make`.

**Comment by kid1412z** [comment]
Check if I'm doing it right.

File: js/jquery.terminal-src.js [hunk 3]
```diff
@@ -1919,7 +1918,7 @@
     var format_last_re = /\[\[[!gbiuso]*;[^;]*;[^\]]*\]?$/i;
     var format_exec_re = /(\[\[(?:[^\]]|\\\])*\]\])/;
     $.terminal = {
-        version: '{{VER}}',
+        version: '0.11.17',
```
Comment (jcubic) [inline js/jquery.terminal-src.js]: Here should be '{{VER}}' because it's source file and it's replaced by version by make.

**Comment by jcubic** [comment]
Thanks for the fix and making the changes.

**Pull request merged**